lib/command.js

```diff
@@ -43,6 +43,11 @@ function Command(name, args, options, callback) {
   this.replyEncoding = options && options.replyEncoding;
   this.errorStack = options && options.errorStack;
   this.args = args ? _.flatten(args) : [];
+  if (options && options.keyPrefix) {
```

Comment:
There's no place we can cache the `keyPrefix` option. `options` is possible to be `undefined` since it's optional. We may put

```javascript
if (typeof options === 'undefined') {
  options = {};
}
```

at the top of the constructor so that we won't need to test options everytime.